src/main/java/org/mskcc/cbio/oncokb/service/MailService.java

```diff
@@ -236,6 +239,30 @@ public void sendTrialAccountExpiresMail(int expiresInDays, List<UserDTO> users)
         }
     }
 
+    @Async
+    public void sendExposedTokensInfoMail(List<ExposedToken> tokens){
+        Context context = new Context(Locale.US);
+        context.setVariable("tokens", tokens);
+        context.setVariable(BASE_URL, jHipsterProperties.getMail().getBaseUrl());
+        String content = templateEngine.process("mail/" + TOKEN_HAS_BEEN_EXPOSED.getTemplateName(), context);
+
+        try{
+            sendEmail(applicationProperties.getEmailAddresses().getTechDevAddress(), applicationProperties.getEmailAddresses().getTechDevAddress(), null, "Exposed tokens", content, null, false, true);
+            log.info("Sent email to User '{}'", applicationProperties.getEmailAddresses().getTechDevAddress());
+        }
+        catch (MailException | MessagingException e){
+            log.warn("Email could not be sent to user '{}'", applicationProperties.getEmailAddresses().getTechDevAddress(), e);
+        }
+    }
+
+    @Async
+    public void sendExposedTokensInfoMailToUser(UserDTO user, ExposedToken token){
```

Comment:
Let's make this name more clear since it's quite long
```suggestion
    public void sendMailToUserWhenTokenExposed(UserDTO user, ExposedToken token){
```